TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/DataAnnotationsExtensions.cs

```diff
@@ -0,0 +1,45 @@
+using System.ComponentModel.DataAnnotations;
```

Comment:
I know it's different to ID but I think putting these into their own `DataAnnotations` namespace makes the naming clearer.

Reply:
Done .. split into separate files too